index.html

```diff
@@ -11,13 +11,12 @@
       rel="stylesheet"
       href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"
     />
+    <script src="javascript.js"></script>
```

Comment:
We shouldn't use javascript as a file name. Please update the file name to something else. Something like empersa.js